Add note in README about availability of most common pairs.

bebosudo opened the pull request:
#123, comment [352266694](https://github.com/nlsdfnbch/bitex/issues/123#issuecomment-352266694).

nlsdfnbch commented on README.md:
````diff
@@ -125,6 +125,13 @@ If we now want to query the rate on, e.g., Bitstamp:
+Note that many common pairs are already implemented and are available inside the main `bitex` module:
+```
+>>> import bitex
+>>> dir(bitex)
````
Import as ' from bitex.pairs import BTCUSD, ETHNIC..' instead - pretty sure some poor soul will try and load it from the list returned by dir() instead of importing it :D

bebosudo replied:
Okay, I just wanted to show where they are located.
Something like:

Note that many common pairs are already implemented and are available inside the main `bitex` module, and can be loaded with `from bitex.pairs import BTCUSD, DASHUSD, ETCUSD, ETHUSD`.
For a complete list, use `dir()` on the main `bitex` module.

Could it be better?

nlsdfnbch replied:
According to the python documentation, `dir` isn't a good choice:

>Because `dir()` is supplied primarily as a convenience for use at an interactive prompt, it tries to supply an interesting set of names more than it tries to supply a rigorously or consistently defined set of names, and **its detailed behavior may change across releases**. For example, metaclass attributes are not in the result list when the argument is a class. 

So it may return the pairs in 3.5 and 3.6, but in 3.7 it might not. Besides, `dir()` lists a whole lot more than the pairs, and sorted alphabetically on top of that.

We either write documentation for this (for example in the module docstring of `pairs.py`, since that is automatically added to the docs at readthedocs.org, or you keep track of the available pairs in the README.md - but I think the former option is better.

bebosudo replied:
Ok for dropping `dir()`.

I agree on writing that in the doc: how would you like to structure it?
Simply listing every pair already implemented (this means that when a new pair is implemented, the doc has to be manually updated, right?), or somewhat "direct" to the `bitex/pairs.py` module (and read the available pairs at the bottom of the file)?

nlsdfnbch replied:
I thing using the autdoc feature __could__ take care of updating it - but the general working would have to be added once manually.

If you look [here](https://github.com/nlsdfnbch/bitex/blob/dev/docs/source/reference.rst), you can see that for the code reference, I don't do much but tell autodoc to list all classes.

So adding a new section for pairs and adding:
```
.. :autodoc:bitex.pairs
   :members:
```

could possibly be all that's required to update the docs. That way, a list is curated by sphinx, and all we have to do is add some brief explanation and example in the `pairs.py` file, along with perhaps a little more detail in the class documentation of the `PairFormatter` objects that have been extended there.

bebosudo replied:
Ok, I'd like to document the available pairs.
How do I setup sphinx (on Fedora linux)? Have you followed any specific tutorial? is there a "sphinx.rc" file to adhere?

nlsdfnbch replied:
you can install it via `pip install sphinx`, then cd into the docs folder should be able to re-build the docs after you made changes with `sphinx-build -b html ./source ./build`. 

You *can* have a look at the `reStructured` syntax if you want to write doc strings with more sophisticated hyperlinking and references to code objects. But it's not mandatory and can be added later as well.

Knock yourself out ! :)

bebosudo commented:
I learned a bit of sphinx, and it was interesting.
I added
```python
autodoc_member_order = 'bysource'
```
into `bitex/docs/source/conf.py` in order to keep the modules in the same order (otherwise the PairFormatter class was mixed up with the implemented pairs).

I also updated the README, removing the reference to `dir`.

I believe some cleanup is required in `bitex/docs/build/_static/`, in particular now there are two jquery libraries inside, maybe the last one can be removed, or maybe both if they are provided by readthedocs.

nlsdfnbch commented:
Great job! Thanks!
